src/calculateRentalCost.js

```diff
@@ -29,6 +29,23 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const basePrice = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const finalPrice = days * basePrice;
```

Comment:
The name is a bit confusing. It is not a final price as it doesn't include discounts